modules/labs-react/layout/stories/examples/StackCard.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {Stack, Flex} from '@workday/canvas-kit-labs-react/layout';
+import {TextButton} from '@workday/canvas-kit-react/button';
+import {type} from '@workday/canvas-kit-labs-react/tokens';
+
+export const StackCard = () => {
+  return (
+    <Flex backgroundColor="soap100" alignItems="flex-start" padding="s">
+      <Flex depth={2} padding="s" backgroundColor="frenchVanilla100">
+        <Stack direction="column" spacing="xs">
+          <h3 style={{...type.h3, margin: 0}}>Stack</h3>
+          <span style={{...type.body}}>
+            Stack provides a simple API for managing consistent space between elements in
+            one-dimensional layouts. In this Card example, we are setting extra-small spacing the
+            heading, body text, and button elements.
+          </span>
+          <Flex>
+            <TextButton>Learn more</TextButton>
+          </Flex>
+        </Stack>
+      </Flex>
+    </Flex>
```

Comment:
The `Flex` components in this example could just as easily be `Box` components, is that right?

I know `Stack` is built on top of `Flex`, but as an end user what are some use cases for when you'd use `Flex` over `Stack`?